<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRFCSFitFunctionSimulator</class>
 <widget class="QDialog" name="QFRDRFCSFitFunctionSimulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FCS Fit Function Simulator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qfrdrfcs.qrc">
    <normaloff>:/fcs_rawdata.png</normaloff>:/fcs_rawdata.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QVisibleHandleSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="JKQtPlotter" name="pltFunction" native="true">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1" columnstretch="0,1" rowminimumheight="0,0,0">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;lag time range:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="buddy">
          <cstring>edtMinTau</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFDoubleEdit" name="edtMinTau"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string> ... </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFDoubleEdit" name="edtMaxTau"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;function:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="buddy">
          <cstring>cmbFunction</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmbFunction"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>parameters:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="layParams"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JKQtPlotter</class>
   <extends>QWidget</extends>
   <header>jkqtplotter.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QFDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qfdoubleedit.h</header>
   <slots>
    <signal>valueChanged(double)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QVisibleHandleSplitter</class>
   <extends>QSplitter</extends>
   <header>qvisiblehandlesplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="qfrdrfcs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFRDRFCSFitFunctionSimulator</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFRDRFCSFitFunctionSimulator</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
